### DISCUSSION TITLE
Welcome to moccasin-full-course-cu Discussions!

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [Cyfrin/moccasin-full-course-cu#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iburr** [comment]
I keep getting this error with titanoboa 

```
"/Users/user/mox-cu/python-in-updraft-cu/basic-python.py", line 1, in <module>
    import boa
ModuleNotFoundError: No module named 'boa'
```
Tried pip and brownie install and everything looks good on my systems installed modules but when it comes to running the basic program itself I keep getting this error.

**Reply by iburr** [reply]
update: I fixed the issue, I was on the wrong Python interpreter. Don't be a noob like me for those reading this lol

**Reply by EngrPips** [reply]
I am glad you figured this out. You are supposed to open a separate discussion for this issue of yours, as this discussion is for getting to know each other.

**Reply by PatrickAlphaC** [reply]
@iburr what did you think of my python explainer where we install python for the first time? Was it confusing?

**Reply by iburr** [reply]
> @iburr what did you think of my python explainer where we install python for the first time? Was it confusing?

Not at all, it was easy to follow through I had just made a mistake. 🤙

**Reply by EngrPips** [reply]
Awesome.

**Comment by ghost** [comment]
# Hello World! :)
My name is _Matthew_, I just started taking the Introduction to **Python and Vyper** course on _Cyfrin Updraft_.

I will be creating a repository to track my progression throughout the course! Please feel free to check it out and provide me feedback!

## Thanks 🙏🏻 Cyfrin Updraft & Patrick for making this course available! ♦️

**Reply by EngrPips** [reply]
Welcome aboard Matthew.

**Reply by Scofield-Idehen** [reply]
welcome mate, i would take some time to check it out.

**Comment by Scofield-Idehen** [comment]
Hello, it's scofield here, I am working and building on Vyper, if you see this down the line, and need help give me a jig on X (scofield_Idehen)

**Comment by iburr** [comment]
Question?: Will the advanced section of the course (Once released) go low-level at all? @PatrickAlphaC

**Reply by EngrPips** [reply]
Hmmmm 🤔

**Comment by wubetlema** [comment]
i am wub  i just learning introduction to Python and Vyper course on Cyfrin Updraft.  how i can take each lesson and discusssion from the beginning

**Reply by EngrPips** [reply]
> i am wub i just learning introduction to Python and Vyper course on Cyfrin Updraft. how i can take each lesson and discusssion from the beginning

Why do you want to take discussion from the beginning.

**Comment by Brian9-king** [comment]
i am experiencing a problem with my vyper code editor where it is not indicating colour changing and tab insertion in the expected areas, its really giving me a hard time

**Reply by s3bc40** [reply]
Hey! 

You can post your own discussion if you want by creating a post with your subject, so everyone can see and help you :smiley: 

Also, do not hesitate to give us screenshots or your code directly (using Markdown syntax). It can help people understand your issue.

If you are using VSCode maybe you might be missing the extension: https://marketplace.visualstudio.com/items?itemName=tintinweb.vscode-vyper

If you search in the marketplace, you should find it :+1: 

![image](https://github.com/user-attachments/assets/cb6b20c1-b87e-4c78-8f67-6c9997428fe4)

**Reply by EngrPips** [reply]
Indeed

**Reply by Brian9-king** [reply]
I appreciate the help guys, hope it's not malicious(:

**Reply by EngrPips** [reply]
I don't think so.

**Reply by Brian9-king** [reply]
unfortunately, it still isn't working...

**Reply by EngrPips** [reply]
Please open a discussion for your issue and provide as much information as possible as @s3bc40 suggested earlier, and in there, we can easily help our

**Reply by Brian9-king** [reply]
I thought i had already opened my discussion, how do i open one?

**Comment by data-fiasco** [comment]
Hello, I am new to this vyper and ethereum. When I tried to deploy buy_me_a_coffee.vy, it got deployed fine, but when trying to send the fund, it gave me the following warning 
"Gas estimation errored with the following message (see below). The transaction execution will likely fail. Do you want to force sending?

missing revert data (action="estimateGas", data=null, reason=null, transaction={ "data": "0xb60d4288", "from": "0x014954aF6e4C8F701370844aC7cbBE3B0Aa686a4", "to": "0x2EbdE42c075d2557FEaDB0481CDD57e99fF222e9" }, invocation=null, revert=null, code=CALL_EXCEPTION, version=6.14.0)"
Despite that when I clicked send, it failed.
AI is not of much help, it is putting m into an infinite loop of errors.

**Reply by s3bc40** [reply]
Hey 👋 

You can open your own topic on the discussion tab so that people can see and help you.

It would be great to add your code from `buy_me_a_coffee.vy`. If you are at the start, are you using Remix IDE? 

Any other information is welcome 👌

**Reply by data-fiasco** [reply]
Yes, I followed all the instructions as it is and am working on Remix ide. The code is also as it is from github, followed each step in the youtube video. In fact I tried opening directly on remix using this link: https://remix.ethereum.org/#url=https://github.com/Cyfrin/remix-buy-me-a-coffee-cu/buy_me_a_coffee.vy
In both cases it gave me same error when I followed the instructions

**Reply by EngrPips** [reply]
Hello @s3bc40, please open a new discussion as suggested by @s3bc40 and share the code you are working on in that new discussion so we can try to reproduce the issue on our end.
